api/v1alpha1/clusterdevfileregistrieslist_webhook.go

```diff
@@ -54,7 +55,7 @@ func (r *ClusterDevfileRegistriesList) Default() {
 var _ webhook.Validator = &ClusterDevfileRegistriesList{}
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
-func (r *ClusterDevfileRegistriesList) ValidateCreate() error {
+func (r *ClusterDevfileRegistriesList) ValidateCreate() (admission.Warnings, error) {
```

Comment:
What is the goal of adding `admission.Warnings` as a return value if we always return `nil` for that field? It is in other functions as well

Reply:
I think is necessary in order to implement the [webhook.Validator](https://github.com/kubernetes-sigs/controller-runtime/blob/c50f2dfd0b618220108d320e3e690340a9246a75/pkg/webhook/admission/validator.go#L37) interface on the `v0.17.5` version of `controller-runtime`

Reply:
We could leave it for these changes and keep in mind that when including warnings to return using this return value. It also states 'The optional warnings will be added to the response as warning messages.' so this is optional in case we provide warnings as well as errors.